scripts/regenerate_test_charts_extended.py

```
@@ -152,6 +154,25 @@ def generate_temporal_subject_charts():
+    for subject_id in future_ids:
+        subject_data = next((s for s in TEMPORAL_SUBJECTS if s["id"] == subject_id), None)
+        if subject_data:
+            try:
+                subject = create_subject_from_dict(subject_data)
+                subject.name = f"{subject_data['name']} - Black-And-White Theme"
+                chart_data = ChartDataFactory.create_natal_chart_data(subject)
+                chart = ChartDrawer(chart_data, theme="black-and-white")
+                chart.save_svg(output_path=OUTPUT_DIR_STR)
+                print(f"  Generated: {subject.name} - Natal Chart - Classic.svg")
+                charts_generated += 1
+            except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)